layers/+lang/scala/config.el

```diff
@@ -11,24 +11,21 @@
 
 (spacemacs|define-jump-handlers scala-mode)
 
-(defvar scala-enable-eldoc nil
-  "If non nil then eldoc-mode is enabled in the scala layer.")
-
 (defvar scala-enable-gtags nil
   "If non nil then gtags is enabled in the scala layer.")
 
 (defvar scala-auto-insert-asterisk-in-comments nil
   "If non-nil automatically insert leading asterisk in multi-line comments.")
 
-(defconst scala-backends '(scala-ensime scala-metals)
+(defconst scala-backends '(scala-metals)
   "Backend server implementation to enable advanced IDE language features")
 
-(defvar scala-backend 'scala-ensime
+(defvar scala-backend 'scala-metals
   "Backend used to trigger IDE language features.
-`scala-ensime' or `scala-metals' are currently supported")
+Only `scala-metals' is currently supported.")
 
 (defvar scala-auto-start-backend nil
```

Comment:
This variable must be removed too, it was only used by ensime, lsp servers are always auto-started.

Reply:
That's why I love deleting. I always learn something. 😁